package.json

```
@@ -22,6 +22,7 @@
   "scripts": {
     "build": "tsc",
     "lint": "npx eslint . --ignore-path .gitignore",
+    "format": "npx prettier . -w",
```

Options should come before paths, in basically all \*nix commands. Also, it's probably useful to use long options for readability:

```suggestion
    "format": "npx prettier --write .",
```